batch_import/README.md

```diff
@@ -73,6 +74,13 @@ In the example above, the Camio Box that's running on port `80` of the ip addres
 Upon completion of the import, all the resulting labeled events can be downloaded using the Camio Search API.
 The downloaded JSON includes the `camera_id`, `camera_name`, `earliest_date`, `latest_date` and `labels` for each event.
 
-TODO(carter) explain the job completion API and bulk download of labeled events.
+### Checking Job Status
+
+@TODO - discuss the Camio API that allows one to query for the status of a job (in progress vs completed). 
```

Comment:
OK to include http://api.camio.com/#jobs now with reference to http://api.camio.com/#get-specific-job